#! /bin/sh -e
## XX_PATCHNAME.dpatch by YOUR NAME <YOUR EMAIL>
##
## All lines beginning with `## DP:' are a description of the patch.
## DP: Fix Bundle Install paths

[ -f debian/patches/00patch-opts ] && . debian/patches/00patch-opts
patch_opts="${patch_opts:--f --no-backup-if-mismatch}"

if [ $# -ne 1 ]; then
	echo >&2 "`basename $0`: script expects -patch|-unpatch as argument"
	exit 1
fi
case "$1" in
	-patch) 
		patch $patch_opts -p1 < $0;;
	-unpatch) 
		patch $patch_opts -p1 -R < $0;;
	*)
		echo >&2 "`basename $0`: script expects -patch|-unpatch as argument"
		exit 1;;
esac

exit 0
@DPATCH@
diff -urN opengroupware.org/Database/OGoModel/GNUmakefile opengroupware.org-latest/Database/OGoModel/GNUmakefile
--- opengroupware.org/Database/OGoModel/GNUmakefile	2004-09-26 18:01:13.000000000 +0200
+++ opengroupware.org-latest/Database/OGoModel/GNUmakefile	2004-09-27 01:24:47.000000000 +0200
@@ -4,7 +4,7 @@
 
 BUNDLE_NAME        = OGo
 BUNDLE_EXTENSION   = .model
-BUNDLE_INSTALL_DIR = $(GNUSTEP_LOCAL_ROOT)/Library/OpenGroupware.org-1.0a/
+BUNDLE_INSTALL_DIR = $(GNUSTEP_INSTALLATION_DIR)/Library/OpenGroupware.org-1.0a/
 
 #ADDITIONAL_CPPFLAGS += -DUSE_EO_RECORDS=1
 
diff -urN opengroupware.org/Logic/LSBase/commandbundle.make opengroupware.org-latest/Logic/LSBase/commandbundle.make
--- opengroupware.org/Logic/LSBase/commandbundle.make	2004-09-26 18:01:12.000000000 +0200
+++ opengroupware.org-latest/Logic/LSBase/commandbundle.make	2004-09-27 01:24:47.000000000 +0200
@@ -7,7 +7,7 @@
 
 BUNDLE_NAME        = $(COMMAND_BUNDLE)
 BUNDLE_EXTENSION   = .cmd
-BUNDLE_INSTALL_DIR = $(GNUSTEP_LOCAL_ROOT)/Library/OpenGroupware.org-1.0a/Commands/
+BUNDLE_INSTALL_DIR = $(GNUSTEP_INSTALLATION_DIR)/Library/OpenGroupware.org-1.0a/Commands/
 
 $(COMMAND_BUNDLE)_RESOURCE_FILES  += commands.plist
 $(COMMAND_BUNDLE)_PRINCIPAL_CLASS = $(COMMAND_BUNDLE)Commands
diff -urN opengroupware.org/Logic/commandbundle.make opengroupware.org-latest/Logic/commandbundle.make
--- opengroupware.org/Logic/commandbundle.make	2004-09-26 18:01:12.000000000 +0200
+++ opengroupware.org-latest/Logic/commandbundle.make	2004-09-27 01:24:47.000000000 +0200
@@ -4,7 +4,7 @@
 
 BUNDLE_NAME        = $(COMMAND_BUNDLE)
 BUNDLE_EXTENSION   = .cmd
-BUNDLE_INSTALL_DIR = $(GNUSTEP_LOCAL_ROOT)/Library/OpenGroupware.org-1.0a/Commands/
+BUNDLE_INSTALL_DIR = $(GNUSTEP_INSTALLATION_DIR)/Library/OpenGroupware.org-1.0a/Commands/
 
 $(COMMAND_BUNDLE)_RESOURCE_FILES  += commands.plist
 $(COMMAND_BUNDLE)_PRINCIPAL_CLASS = $(COMMAND_BUNDLE)Commands
diff -urN opengroupware.org/Misc/OGoLDAPClientUI/GNUmakefile opengroupware.org-latest/Misc/OGoLDAPClientUI/GNUmakefile
--- opengroupware.org/Misc/OGoLDAPClientUI/GNUmakefile	2004-09-26 18:01:01.000000000 +0200
+++ opengroupware.org-latest/Misc/OGoLDAPClientUI/GNUmakefile	2004-09-27 01:24:47.000000000 +0200
@@ -4,7 +4,7 @@
 
 WOBUNDLE_NAME        = OGoLDAPClientUI
 WOBUNDLE_EXTENSION   = .lso
-WOBUNDLE_INSTALL_DIR = $(GNUSTEP_LOCAL_ROOT)/Library/OpenGroupware.org
+WOBUNDLE_INSTALL_DIR = $(GNUSTEP_INSTALLATION_DIR)/Library/OpenGroupware.org
 
 OGoLDAPClientUI_PRINCIPAL_CLASS = LdapViewerModule
 
diff -urN opengroupware.org/PDA/OGoNHSSync/GNUmakefile opengroupware.org-latest/PDA/OGoNHSSync/GNUmakefile
--- opengroupware.org/PDA/OGoNHSSync/GNUmakefile	2004-09-26 18:01:14.000000000 +0200
+++ opengroupware.org-latest/PDA/OGoNHSSync/GNUmakefile	2004-09-27 01:25:53.000000000 +0200
@@ -20,7 +20,7 @@
 
 BUNDLE_NAME        = OpenGroupwareNHS
 BUNDLE_EXTENSION   = .conduit
-BUNDLE_INSTALL_DIR = $(GNUSTEP_USER_ROOT)/Library/OpenGroupware.org-1.0a
+BUNDLE_INSTALL_DIR = $(GNUSTEP_INSTALLATION_DIR)/Library/OpenGroupware.org-1.0a
 
 OpenGroupwareNHS_OBJC_FILES = \
 	OGoNHSConduit.m	\
diff -urN opengroupware.org/PDA/OGoPalm/GNUmakefile opengroupware.org-latest/PDA/OGoPalm/GNUmakefile
--- opengroupware.org/PDA/OGoPalm/GNUmakefile	2004-09-26 18:01:14.000000000 +0200
+++ opengroupware.org-latest/PDA/OGoPalm/GNUmakefile	2004-09-27 01:25:13.000000000 +0200
@@ -9,7 +9,7 @@
 LIBRARY_NAME       = libOGoPalm
 BUNDLE_NAME        = OGoPalmDS
 BUNDLE_EXTENSION   = .ds
-BUNDLE_INSTALL_DIR = $(GNUSTEP_LOCAL_ROOT)/Library/OpenGroupware.org-1.0a/DataSources/
+BUNDLE_INSTALL_DIR = $(GNUSTEP_INSTALLATION_DIR)/Library/OpenGroupware.org-1.0a/DataSources/
 
 libOGoPalm_SOVERSION=$(MAJOR_VERSION).$(MINOR_VERSION)
 libOGoPalm_VERSION=$(MAJOR_VERSION).$(MINOR_VERSION).$(SUBMINOR_VERSION)
diff -urN opengroupware.org/PDA/OGoPalmWebUI/GNUmakefile opengroupware.org-latest/PDA/OGoPalmWebUI/GNUmakefile
--- opengroupware.org/PDA/OGoPalmWebUI/GNUmakefile	2004-09-26 18:01:14.000000000 +0200
+++ opengroupware.org-latest/PDA/OGoPalmWebUI/GNUmakefile	2004-09-27 01:24:47.000000000 +0200
@@ -8,7 +8,7 @@
 
 WOBUNDLE_NAME        = OGoPalm
 WOBUNDLE_EXTENSION   = .lso
-WOBUNDLE_INSTALL_DIR = $(GNUSTEP_USER_ROOT)/Library/OpenGroupware.org-1.0a/WebUI/
+WOBUNDLE_INSTALL_DIR = $(GNUSTEP_INSTALLATION_DIR)/Library/OpenGroupware.org-1.0a/WebUI/
 
 OGoPalm_PRINCIPAL_CLASS = SkyPalmModule
 
diff -urN opengroupware.org/Themes/WebServerResources/GNUmakefile opengroupware.org-latest/Themes/WebServerResources/GNUmakefile
--- opengroupware.org/Themes/WebServerResources/GNUmakefile	2004-09-26 18:01:00.000000000 +0200
+++ opengroupware.org-latest/Themes/WebServerResources/GNUmakefile	2004-09-27 01:24:47.000000000 +0200
@@ -1,7 +1,7 @@
 # $Id$
 
 WEBSERVER_RESOURCES_INSTALLATION_DIR = \
-		$(GNUSTEP_LOCAL_ROOT)/WebServerResources
+		$(GNUSTEP_INSTALLATION_DIR)/WebServerResources
 
 Skyrix_LANGUAGE_Projects = \
 	English.lproj		\
diff -urN opengroupware.org/WebUI/AdminUI/GNUmakefile opengroupware.org-latest/WebUI/AdminUI/GNUmakefile
--- opengroupware.org/WebUI/AdminUI/GNUmakefile	2004-09-26 18:01:22.000000000 +0200
+++ opengroupware.org-latest/WebUI/AdminUI/GNUmakefile	2004-09-27 01:24:47.000000000 +0200
@@ -4,7 +4,7 @@
 
 WOBUNDLE_NAME        = AdminUI
 WOBUNDLE_EXTENSION   = .lso
-WOBUNDLE_INSTALL_DIR = $(GNUSTEP_LOCAL_ROOT)/Library/OpenGroupware.org-1.0a/WebUI
+WOBUNDLE_INSTALL_DIR = $(GNUSTEP_INSTALLATION_DIR)/Library/OpenGroupware.org-1.0a/WebUI
 
 AdminUI_PRINCIPAL_CLASS = LSWUserManagerModule
 
diff -urN opengroupware.org/WebUI/HelpUI/GNUmakefile opengroupware.org-latest/WebUI/HelpUI/GNUmakefile
--- opengroupware.org/WebUI/HelpUI/GNUmakefile	2004-09-26 18:01:20.000000000 +0200
+++ opengroupware.org-latest/WebUI/HelpUI/GNUmakefile	2004-09-27 01:24:47.000000000 +0200
@@ -4,7 +4,7 @@
 
 WOBUNDLE_NAME        = HelpUI
 WOBUNDLE_EXTENSION   = .lso
-WOBUNDLE_INSTALL_DIR = $(GNUSTEP_LOCAL_ROOT)/Library/OpenGroupware.org-1.0a/WebUI/
+WOBUNDLE_INSTALL_DIR = $(GNUSTEP_INSTALLATION_DIR)/Library/OpenGroupware.org-1.0a/WebUI/
 
 HelpUI_PRINCIPAL_CLASS = LSWJobModule
 
diff -urN opengroupware.org/WebUI/JobUI/GNUmakefile opengroupware.org-latest/WebUI/JobUI/GNUmakefile
--- opengroupware.org/WebUI/JobUI/GNUmakefile	2004-09-26 18:01:27.000000000 +0200
+++ opengroupware.org-latest/WebUI/JobUI/GNUmakefile	2004-09-27 01:24:47.000000000 +0200
@@ -4,7 +4,7 @@
 
 WOBUNDLE_NAME        = JobUI
 WOBUNDLE_EXTENSION   = .lso
-WOBUNDLE_INSTALL_DIR = $(GNUSTEP_LOCAL_ROOT)/Library/OpenGroupware.org-1.0a/WebUI/
+WOBUNDLE_INSTALL_DIR = $(GNUSTEP_INSTALLATION_DIR)/Library/OpenGroupware.org-1.0a/WebUI/
 
 JobUI_PRINCIPAL_CLASS = JobUIModule
 
diff -urN opengroupware.org/WebUI/Mailer/common.make opengroupware.org-latest/WebUI/Mailer/common.make
--- opengroupware.org/WebUI/Mailer/common.make	2004-09-26 18:01:48.000000000 +0200
+++ opengroupware.org-latest/WebUI/Mailer/common.make	2004-09-27 01:24:47.000000000 +0200
@@ -7,7 +7,7 @@
 include $(WebUIROOT)/Version
 
 WOBUNDLE_EXTENSION   = .lso
-WOBUNDLE_INSTALL_DIR = $(GNUSTEP_LOCAL_ROOT)/Library/OpenGroupware.org-1.0a/WebUI/
+WOBUNDLE_INSTALL_DIR = $(GNUSTEP_INSTALLATION_DIR)/Library/OpenGroupware.org-1.0a/WebUI/
 
 ADDITIONAL_INCLUDE_DIRS += \
 	-I.. -I$(WebUIROOT)\
diff -urN opengroupware.org/WebUI/NewsUI/GNUmakefile opengroupware.org-latest/WebUI/NewsUI/GNUmakefile
--- opengroupware.org/WebUI/NewsUI/GNUmakefile	2004-09-26 18:01:48.000000000 +0200
+++ opengroupware.org-latest/WebUI/NewsUI/GNUmakefile	2004-09-27 01:24:47.000000000 +0200
@@ -4,7 +4,7 @@
 
 WOBUNDLE_NAME        = NewsUI
 WOBUNDLE_EXTENSION   = .lso
-WOBUNDLE_INSTALL_DIR = $(GNUSTEP_LOCAL_ROOT)/Library/OpenGroupware.org-1.0a/WebUI
+WOBUNDLE_INSTALL_DIR = $(GNUSTEP_INSTALLATION_DIR)/Library/OpenGroupware.org-1.0a/WebUI
 
 NewsUI_PRINCIPAL_CLASS = LSWNewsModule
 
diff -urN opengroupware.org/WebUI/PreferencesUI/GNUmakefile opengroupware.org-latest/WebUI/PreferencesUI/GNUmakefile
--- opengroupware.org/WebUI/PreferencesUI/GNUmakefile	2004-09-26 18:01:27.000000000 +0200
+++ opengroupware.org-latest/WebUI/PreferencesUI/GNUmakefile	2004-09-27 01:24:47.000000000 +0200
@@ -6,7 +6,7 @@
 
 WOBUNDLE_NAME        = PreferencesUI
 WOBUNDLE_EXTENSION   = .lso
-WOBUNDLE_INSTALL_DIR = $(GNUSTEP_LOCAL_ROOT)/Library/OpenGroupware.org-1.0a/WebUI/
+WOBUNDLE_INSTALL_DIR = $(GNUSTEP_INSTALLATION_DIR)/Library/OpenGroupware.org-1.0a/WebUI/
 
 PreferencesUI_PRINCIPAL_CLASS = LSWPreferencesModule
 
diff -urN opengroupware.org/WebUI/Project/common.make opengroupware.org-latest/WebUI/Project/common.make
--- opengroupware.org/WebUI/Project/common.make	2004-09-26 18:01:27.000000000 +0200
+++ opengroupware.org-latest/WebUI/Project/common.make	2004-09-27 01:24:47.000000000 +0200
@@ -7,7 +7,7 @@
 include $(WebUIROOT)/Version
 
 WOBUNDLE_EXTENSION   = .lso
-WOBUNDLE_INSTALL_DIR = $(GNUSTEP_LOCAL_ROOT)/Library/OpenGroupware.org-1.0a/WebUI/
+WOBUNDLE_INSTALL_DIR = $(GNUSTEP_INSTALLATION_DIR)/Library/OpenGroupware.org-1.0a/WebUI/
 
 ADDITIONAL_INCLUDE_DIRS += \
 	-I.. -I../..		\
diff -urN opengroupware.org/WebUI/Resources/GNUmakefile opengroupware.org-latest/WebUI/Resources/GNUmakefile
--- opengroupware.org/WebUI/Resources/GNUmakefile	2004-09-26 18:01:50.000000000 +0200
+++ opengroupware.org-latest/WebUI/Resources/GNUmakefile	2004-09-27 01:24:47.000000000 +0200
@@ -1,7 +1,7 @@
 # $Id$
 
 RESOURCES_INSTALLATION_DIR = \
-		$(GNUSTEP_LOCAL_ROOT)/Resources
+		$(GNUSTEP_INSTALLATION_DIR)/Resources
 
 Skyrix_LANGUAGE_Projects = \
 	English.lproj 		\
diff -urN opengroupware.org/WebUI/SoOGo/GNUmakefile opengroupware.org-latest/WebUI/SoOGo/GNUmakefile
--- opengroupware.org/WebUI/SoOGo/GNUmakefile	2004-09-26 18:01:20.000000000 +0200
+++ opengroupware.org-latest/WebUI/SoOGo/GNUmakefile	2004-09-27 01:24:47.000000000 +0200
@@ -4,7 +4,7 @@
 
 WOBUNDLE_NAME        = SoOGo
 WOBUNDLE_EXTENSION   = .lso
-WOBUNDLE_INSTALL_DIR = $(GNUSTEP_LOCAL_ROOT)/Library/OpenGroupware.org-1.0a/WebUI/
+WOBUNDLE_INSTALL_DIR = $(GNUSTEP_INSTALLATION_DIR)/Library/OpenGroupware.org-1.0a/WebUI/
 
 SoOGo_PRINCIPAL_CLASS = SoOGoModule
 
diff -urN opengroupware.org/ZideStore/PrefsUI/GNUmakefile opengroupware.org-latest/ZideStore/PrefsUI/GNUmakefile
--- opengroupware.org/ZideStore/PrefsUI/GNUmakefile	2004-09-26 18:01:17.000000000 +0200
+++ opengroupware.org-latest/ZideStore/PrefsUI/GNUmakefile	2004-09-27 01:24:47.000000000 +0200
@@ -6,7 +6,7 @@
 
 BUNDLE_NAME        = PrefsUI
 BUNDLE_EXTENSION   = .zsp
-BUNDLE_INSTALL_DIR = $(GNUSTEP_USER_ROOT)/Library/ZideStore-$(MAJOR_VERSION).$(MINOR_VERSION)
+BUNDLE_INSTALL_DIR = $(GNUSTEP_INSTALLATION_DIR)/Library/ZideStore-$(MAJOR_VERSION).$(MINOR_VERSION)
 
 PrefsUI_PRINCIPAL_CLASS = PrefsUIProduct
 
diff -urN opengroupware.org/ZideStore/Protocols/EvoConnect/GNUmakefile opengroupware.org-latest/ZideStore/Protocols/EvoConnect/GNUmakefile
--- opengroupware.org/ZideStore/Protocols/EvoConnect/GNUmakefile	2004-09-26 18:01:16.000000000 +0200
+++ opengroupware.org-latest/ZideStore/Protocols/EvoConnect/GNUmakefile	2004-09-27 01:24:47.000000000 +0200
@@ -6,7 +6,7 @@
 
 BUNDLE_NAME        = EvoConnect
 BUNDLE_EXTENSION   = .zsp
-BUNDLE_INSTALL_DIR = $(GNUSTEP_USER_ROOT)/Library/ZideStore-$(MAJOR_VERSION).$(MINOR_VERSION)
+BUNDLE_INSTALL_DIR = $(GNUSTEP_INSTALLATION_DIR)/Library/ZideStore-$(MAJOR_VERSION).$(MINOR_VERSION)
 
 EvoConnect_PRINCIPAL_CLASS = EvoConnectProduct
 
diff -urN opengroupware.org/ZideStore/Protocols/RSS/GNUmakefile opengroupware.org-latest/ZideStore/Protocols/RSS/GNUmakefile
--- opengroupware.org/ZideStore/Protocols/RSS/GNUmakefile	2004-09-26 18:01:16.000000000 +0200
+++ opengroupware.org-latest/ZideStore/Protocols/RSS/GNUmakefile	2004-09-27 01:24:47.000000000 +0200
@@ -6,7 +6,7 @@
 
 BUNDLE_NAME        = RSS
 BUNDLE_EXTENSION   = .zsp
-BUNDLE_INSTALL_DIR = $(GNUSTEP_USER_ROOT)/Library/ZideStore-$(MAJOR_VERSION).$(MINOR_VERSION)
+BUNDLE_INSTALL_DIR = $(GNUSTEP_INSTALLATION_DIR)/Library/ZideStore-$(MAJOR_VERSION).$(MINOR_VERSION)
 
 RSS_PRINCIPAL_CLASS = RSSProduct
 
diff -urN opengroupware.org/ZideStore/Protocols/WCAP/GNUmakefile opengroupware.org-latest/ZideStore/Protocols/WCAP/GNUmakefile
--- opengroupware.org/ZideStore/Protocols/WCAP/GNUmakefile	2004-09-26 18:01:16.000000000 +0200
+++ opengroupware.org-latest/ZideStore/Protocols/WCAP/GNUmakefile	2004-09-27 01:24:47.000000000 +0200
@@ -6,7 +6,7 @@
 
 BUNDLE_NAME        = WCAP
 BUNDLE_EXTENSION   = .zsp
-BUNDLE_INSTALL_DIR = $(GNUSTEP_USER_ROOT)/Library/ZideStore-$(MAJOR_VERSION).$(MINOR_VERSION)
+BUNDLE_INSTALL_DIR = $(GNUSTEP_INSTALLATION_DIR)/Library/ZideStore-$(MAJOR_VERSION).$(MINOR_VERSION)
 
 WCAP_PRINCIPAL_CLASS = WCAPProduct
 
diff -urN opengroupware.org/ZideStore/SoObjects/common.make opengroupware.org-latest/ZideStore/SoObjects/common.make
--- opengroupware.org/ZideStore/SoObjects/common.make	2004-09-26 18:01:16.000000000 +0200
+++ opengroupware.org-latest/ZideStore/SoObjects/common.make	2004-09-27 01:24:47.000000000 +0200
@@ -8,7 +8,7 @@
 
 
 BUNDLE_EXTENSION   = .zsp
-BUNDLE_INSTALL_DIR = $(GNUSTEP_USER_ROOT)/Library/ZideStore-$(MAJOR_VERSION).$(MINOR_VERSION)/
+BUNDLE_INSTALL_DIR = $(GNUSTEP_INSTALLATION_DIR)/Library/ZideStore-$(MAJOR_VERSION).$(MINOR_VERSION)/
 
 
 ADDITIONAL_INCLUDE_DIRS += -I. -I.. -I../..

